python/Ganga/new_tests/TestProxy.py

```diff
@@ -79,7 +85,7 @@ def test_dict_attributes(self):
         self.assertFalse(hasattr(proxied, '_impl'))
 
         # A proxy instance should only have _impl in its dict
-        self.assertEqual(self.p.__dict__.keys(), ['_impl'])
+        self.assertEqual(self.p.__dict__.keys(), ['_impl', '_proxyObject'])
```

Comment:
No, a proxy object should *never* have a `_proxyObject` attribute. A proxy should just point to its implementation. Nothing should ever add one and if it does then it's wrong.

It used to be different (before my recent changes) but now a proxy object never point to itself (or anywhere) with `_proxyObject`. See http://ganga.readthedocs.org/en/latest/dev/index.html#attributes-on-proxy-objects